examples/resources/cloudavenue_iam_user/import.sh

```diff
@@ -0,0 +1,2 @@
+# use the user-name to import the User
+terraform import cloudavenue_iam_user.example exampleuser
```

Comment:
why you don't use name like other example of import ?

terraform import cloudavenue_iam_user.example name